data/index.js

```diff
@@ -1,7 +1,7 @@
 var self = require('sdk/self');
 
-// a dummy function, to show how tests work. 
-// to see how to test this function, look at ../test/test-main.js
+// a dummy function, to show how tests work.
+// to see how to test this function, look at test/test-index.js
```

Comment:
Right, it's used in the tests, but that doesn't modify the directory structure at any point, so `index.js` is always a sibling of the `test` directory and getting to `test-index.js` would just be `./test/test-index.js`, no? I think I'm missing something here. Happy to change it if you insist, but I don't see any scenario where `test` is up a directory.